### DISCUSSION TITLE
Add "issues", please!

### DISCUSSION THREAD

**Discussion by wujastyk** [PanditProject/Website#21] [Help] [answered]
Dear Amir, please, please would you activate the "issues" feature of Github for this repo.  It's one of the principle strengths of Github.  I'm certain that once you get used to it you will find it valuable.

I hope you can!

Best,
Dominik

**Comment by AmirSimantov** [comment]
Dear @wujastyk, could you please specify what are the difficulties with **Discussions** and what you think that **Issues** does better?

**Comment by wujastyk** [comment]
Both have their place.  "Issues" are uniquely useful because they are
oriented to specific, focussed tasks that someone has to do.  They permit
assigning a task to an individual person or persons, and flagging the task
in useful ways including whether it is a critical problem, or a feature
suggestion, etc., and when it is completed. Tasks can be ranked in
importance, and can refer to each other.

Discussion is excellent for exploring topics that are not connected with
tasks that have to be done by somebody.

Both are good.

Always choose the right tool for the job. :-)

Best,
Dominik

**Reply by AmirSimantov** [reply]
Dear @wujastyk 

"Issues" are uniquely helpful because they are oriented to specific, focussed tasks that someone has to do."
- We have decided to work on GitHub for transparency and collaboration; we don't use it as a task management system. And anyway, we are not in a position to assign tasks to people; this isn't the way Pandit works, for now.

"and flagging the task in useful ways including whether it is a critical problem or a feature suggestion, etc., and when it is completed."
There are also labels for Discussions; we will use them once we have many discussions; we are trying to simplify it. We can use the labels in any way we want, including priority or category. When we finish a discussion, I assign one of the threads of a discussion as an answer, and you then can filter the discussions by Answered/Not answered (what you call "completed").

"Tasks can be ranked in importance and can refer to each other"
- We don't have so many discussions; if we do, we can use the labels feature. You can refer to another discussion just as you do with Issues; please use "#" and then choose the one you want to refer to from the discussion list.

"Discussion is excellent for exploring topics that are not connected with tasks that have to be done by somebody"
- Again, we don't assign a contributor with a task; we don't feel we have the mandate for that now. Instead, we discuss, raise ideas, brainstorm, etc.

The **Discussions** feature is very new to GitHub (just released about half a year ago), and I understand it takes time to get used to it. As we are in the transition of working through emails, which are also a way to discuss among us, to work in a more public way, the Discussions feature may well be what we need and try to encourage.

Having said that, when we get to plan Pandit version 2, we WILL use the Issues feature. We will then have potentially many many of them (what feature to improve, what feature is missing, what existing feature we don't use, user experience issues, design, etc.).

**Comment by wujastyk** [comment]
Please don't say no.  I would be grateful if you would reconsider and open
up issues.  Let's just be experimental!  If we don't like it, we don't have
to continue.  If it has some utility for certain kinds of discussion, we
can use it.

It's not an issue of "getting used to" Discussions.  I'm perfectly familiar
with such lists, and I run several myself.  They are the right tool for
certain types of exchange.  They are not the best tool when the discussion
moves into task management, for example fixing things, checking
functionality, etc., all issue that have arisen many times in our so-called
discussion.

So I press you again.  Please stop disabling "issues."


Best,
Dominik

**Comment by ybronner** [comment, marked as answer]
Dear Dominik, we are by no means saying "no." We will have a dedicated issues reporting system. The question is only the timing. We are in the process of creating Pandit 2, on Drupal 9, which will replace the current one. We expect it to be in place within the next 12 months. So it makes little sense now to open a long list of issues that will be mostly relevant for a system that will soon be outdated. Once Pandit 2 is in place, we will immediately start issues, and we will actively invite people to report all kinds of issues, bugs, and tasks. For now, our energy and resources are limited and are dedicated to this overhaul, and we would much rather limit ourselves to more general discussions (that might also help us in the process).
